### ISSUE TITLE
protocArtifact and pluginArtifact pull artifacts that can't be executed on Mac OS

### ISSUE THREAD

**Issue by kleinsch** [os72/protoc-jar-maven-plugin#32]
I'm using Maven 3.3.9. It looks like Maven dependencies get downloaded without the execute bit set, so the plugin fails when it tries to execute something pulled from Maven.

```
Unable to execute protoc-jar for my.proto: Cannot run program "/Users/nick/.m2/repository/com/google/protobuf/protoc/3.1.0/protoc-3.1.0-osx-x86_64.exe": error=13, Permission denied -> [Help 1]
```

```
Nicks-MacBook-Pro:3.1.0 nick$ pwd
/Users/nick/.m2/repository/com/google/protobuf/protoc/3.1.0
Nicks-MacBook-Pro:3.1.0 nick$ ls -l
total 9672
-rw-r--r--  1 nick  staff      170 Jan  9 10:19 _remote.repositories
-rw-r--r--  1 nick  staff  4943484 Jan  9 10:19 protoc-3.1.0-osx-x86_64.exe
-rw-r--r--  1 nick  staff       40 Jan  9 10:19 protoc-3.1.0-osx-x86_64.exe.sha1

```

```
/Users/nick/.m2/repository/io/grpc/protoc-gen-grpc-java/1.0.3/protoc-gen-grpc-java-1.0.3-osx-x86_64.exe: program not found or is not executable
--grpc-java_out: protoc-gen-grpc-java: Plugin failed with status code 1.
```

```
Nicks-MacBook-Pro:1.0.3 nick$ pwd
/Users/nick/.m2/repository/io/grpc/protoc-gen-grpc-java/1.0.3
Nicks-MacBook-Pro:1.0.3 nick$ ls -l
total 4216
-rw-r--r--  1 nick  staff      184 Jan  9 10:16 _remote.repositories
-rw-r--r--  1 nick  staff  2149556 Jan  9 10:16 protoc-gen-grpc-java-1.0.3-osx-x86_64.exe
-rw-r--r--  1 nick  staff       40 Jan  9 10:16 protoc-gen-grpc-java-1.0.3-osx-x86_64.exe.sha1
```

**Comment by os72** [comment]
Ack

**Comment by mindictor** [comment]
Any idea when this might get fixed?

**Comment by os72** [comment]
May take a few days till I can get to it

**Comment by os72** [comment]
Released 3.1.0.4
